### ISSUE TITLE
add priority in scheduling for the common operator

### ISSUE THREAD

**Issue by YesterdayxD** [kubeflow/common#46]
hi，everyone:
I find that it has a attribution about PodGroupSpec.PriorityClassName in kubebatch, so I want to add this attribution into the operators of kubeflow so that it is uesd by kubebatch.
I try to add PriorityClassName into RunPolicy.SchedulingPolicy.
`// SchedulingPolicy encapsulates various scheduling policies of the distributed training
// job, for example `minAvailable` for gang-scheduling.
type SchedulingPolicy struct {
	MinAvailable *int32 `json:"minAvailable,omitempty"`

	//PriorityClassName is a type of k8s resource.(kubectl get priorityclass)
	PriorityClassName *string `json:"priorityClassName,omitempty"`
}` 
when kubebatch need the priority,it can get it from RunPolicy.

**Comment by k82cn** [comment]
Are we going to inline `SchedulingPolicy` in `PyTorchJobSpec`? What's the suggestion to other operator?

**Comment by davidstack** [comment]
maybe all operrator should add SchedulingPolicy,so we can add SchedulingPolicy  in this comnon package

**Comment by gaocegege** [comment]
Yeah, we should add SchedulingPolicy to common. But pytorch-operator and tf-operator does not use common now. We should re-implement the logic in these operators, too

**Comment by 4everming** [comment]
Yes, we should implement the logic in MXNet-Operator too.

**Comment by gaocegege** [comment]
@johnugeorge @richardsliu 

Do you have any suggestion?

**Comment by YesterdayxD** [comment]
retire wg-machine-learning？It is so bad.

**Comment by k82cn** [comment]
> retire wg-machine-learning？It is so bad.

Nop for now, I'll help to maintain ML-WG for a while; if still not working items, we'll retire it :)

**Comment by gaocegege** [comment]
Hi is there any update?

**Comment by k82cn** [comment]
hm... are we going to do this feature?

**Comment by terrytangyuan** [comment]
Yes it's part of our roadmap so contribution is welcomed.

**Comment by kerthcet** [comment]
I think we can close this refer to https://github.com/kubeflow/common/blob/21f5ba8833a2e21df17601497a08396c9bae9ab2/pkg/apis/common/v1/types.go#L204-L209
